aws_lambda_powertools/utilities/idempotency/base.py

```diff
@@ -65,12 +72,17 @@ def __init__(
             Idempotency Configuration
         persistence_store : BasePersistenceLayer
             Instance of persistence layer to store idempotency records
+        output_serializer: Optional[BaseDictSerializer]
```

Comment:
```suggestion
        output_serializer: BaseDictSerializer
```

Reply:
Currently the class allows to not pass 'output_serializer' parameter.
Is this change correct?

Reply:
I reconsidered :D Your current code looks great, so ignore this suggestion.